src/modules/75-cf/pages/onboarding/OnboardingPage.tsx

```diff
@@ -25,30 +25,37 @@ export const OnboardingPage = (): React.ReactElement => {
   const { trackEvent } = useTelemetry()
 
   return (
-    <Container>
+    <>
+      {/* Products free banner */}
       <Layout.Horizontal flex={{ justifyContent: 'center' }} width="100%" margin={{ bottom: 'huge' }}>
         <img src={productsFreeForever} />
       </Layout.Horizontal>
-      <Container padding="huge" flex={{ justifyContent: 'center' }} height="65vh">
-        <Layout.Horizontal height="auto" width="auto" className={css.mainContentContainer} padding="huge">
-          {/* Left side - Text & Button */}
-          <Layout.Vertical flex={{ alignItems: 'flex-start', justifyContent: 'space-between' }}>
+
+      {/* Overall Layout */}
+      <Container padding="huge" flex={{ justifyContent: 'center' }} height="650px">
+        <Layout.Horizontal
+          flex={{ justifyContent: 'center' }}
+          height="auto"
+          width="auto"
+          className={css.mainContentContainer}
+          padding="huge"
+        >
+          {/* Left Side */}
+          <Container className={css.leftGrid}>
             <Layout.Horizontal spacing="small" style={{ justifyContent: 'flex-start', alignItems: 'center' }}>
               <Icon name="cf-main" height={30} size={30} />
               <Text font={{ variation: FontVariation.SMALL_BOLD }} color={Color.GREY_800}>
                 {getString('common.purpose.cf.continuous')}
               </Text>
             </Layout.Horizontal>
-            <Layout.Vertical>
+            <Layout.Vertical flex={{ alignItems: 'flex-start', justifyContent: 'space-between' }}>
               <Text font={{ variation: FontVariation.H2 }} tag="h2" margin={{ bottom: 'medium' }}>
                 {getString('cf.onboarding.title')}
               </Text>
               <Text font={{ variation: FontVariation.BODY }} color={Color.GREY_800} margin={{ bottom: 'huge' }}>
                 {getString('cf.onboarding.subTitle')}
               </Text>
               {OnboardingStepsDescription()}
```

Comment:
Why is this a function call instead of a component? Weird
```suggestion
              <OnboardingStepsDescription />
```